### PR TITLE
Fixed #34568 -- Made makemigrations --update respect --name option.

### PR DESCRIPTION
**Pull request by felixxm**
Thanks David Sanders for the report.

ticket-34568

### CODE REVIEW

**Review by shangxiao — APPROVED** [review]
Looks good👍

Are we updating the docs as part of this to mention that the name isn't preserved, instead auto generated per usual and to keep the name one must use --name ?

Edit: isn't preserved

**Comment by felixxm** [comment]
> Are we updating the docs as part of this to mention that the name isn't preserved, instead auto generated per usual and to keep the name one must use --name ?
> 
> Edit: isn't preserved

I added a short note.

File: docs/ref/django-admin.txt [hunk 1]
```diff
@@ -757,6 +757,9 @@ generated migration files to ``stdout``.
 Merges model changes into the latest migration and optimize the resulting
 operations.
 
+Updated migration has a generated name. You can use ``--name`` to preserve the
+previous name.
```
Comment (shangxiao) [inline docs/ref/django-admin.txt:761]:
```suggestion
The updated migration will have a generated name; in order to preserve the existing name you must set it using ``--name``.
```

Suggestion^^ I think the wording flows better, others may have better wording 🤷‍♂️

Comment (felixxm) [reply docs/ref/django-admin.txt:761]: Updated, but not with exact wording :wink:

**Comment by felixxm** [comment]
@shangxiao Thanks for checking :+1:

**Pull request merged**